ansible_collections/arista/avd/plugins/action/eos_designs_facts.py

```diff
@@ -5,15 +5,11 @@
 import pstats
 from ansible.errors import AnsibleActionFail
 from ansible.plugins.action import ActionBase
-from ansible.plugins.filter.core import combine
 from ansible.plugins.loader import lookup_loader
 from ansible.template import Templar
-from ansible_collections.arista.avd.plugins.filter.convert_dicts import convert_dicts
-from ansible_collections.arista.avd.plugins.filter.list_compress import list_compress
-from ansible_collections.arista.avd.plugins.filter.range_expand import range_expand
-from ansible_collections.arista.avd.plugins.filter.natural_sort import natural_sort
 from ansible_collections.arista.avd.plugins.module_utils.eos_designs_facts import EosDesignsFacts
 from ansible_collections.arista.avd.plugins.module_utils.utils import AristaAvdMissingVariableError
+from ansible.plugins.filter.core import combine
```

Comment:
lets move this back to its original line (line 8)

Reply:
Thanks for the review @gmuloc , I will look into your comments today.

Reply:
Please rebase. We no longer use this combine. Instead we use our own merge function.

Reply:
yes, I have rebased it. I can see we are still having combine in eos_designs_to_facts , shall I refactor it in this PR?